parsl/jobs/strategy.py

```diff
@@ -146,10 +146,17 @@ def add_executors(self, executors: Sequence[ParslExecutor]) -> None:
         for executor in executors:
             self.executors[executor.label] = {'idle_since': None}
 
-    def _strategy_noop(self, status: List[jsp.PollItem]) -> None:
-        """Do nothing.
+    def _strategy_init_only(self, status_list: List[jsp.PollItem]) -> None:
+        """Scale up to init_blocks at the start, then nothing more.
         """
-        logger.debug("strategy_noop: doing nothing")
+        for exec_status in status_list:
+            if exec_status.first:
+                executor = exec_status.executor
+                logger.debug(f"strategy_init_only: scaling out {executor.provider.init_blocks} initial blocks for {executor.label}")
+                exec_status.scale_out(executor.provider.init_blocks)
```

Comment:
For new SLOCs, might consider passing the args to `.debug()` rather that doing the formatting up front.  No sense in doing the work if debug-level logs aren't emitted:
```python
logger.debug("scaling out %s initial blocks for %s", executor.provider.init_blocks, executor.label)
```

Reply:
this isn't a new line - it's a move and a keyword rename!

Reply:
:boom: